extensions/datasource/deployment-spi/src/main/java/io/quarkus/datasource/deployment/spi/DevServicesUseTestContainersSharedNetworkBuildItem.java

```diff
@@ -0,0 +1,10 @@
+package io.quarkus.datasource.deployment.spi;
+
+import io.quarkus.builder.item.SimpleBuildItem;
+
+/**
+ * Marker used by Build Steps that launch TestContainers' containers to ensure that they join TestContainers shared network.
+ */
+public final class DevServicesUseTestContainersSharedNetworkBuildItem extends SimpleBuildItem {
```

Comment:
This needs to be a `MultiBuildItem` because it could theoretically be generated by multiple extensions

Reply:
Updated.

Reply:
I was thinking @geoand.

Would you like me to also update the other `BuildStep`'s that currently join TestContainers shared network. See the list I `grep`'ed [here](https://github.com/quarkusio/quarkus/pull/21776#discussion_r758484750) as part of this PR? Or did you want to keep my change limited to the use case I have?

Reply:
That's a good idea. Please do that